ESP/platformio.ini

```diff
@@ -9,7 +9,35 @@
 ; https://docs.platformio.org/page/projectconf.html
 
 [platformio]
-default_envs = debug
+default_envs = debug ; do not change this value
+
+; The below options are available for all environments
+; The ssid and password are requried for the trackers to connect to your network!!!
+[wifi]
+ssid="your_ssid_goes_here" ; your wifi network name goes here
```

Comment:
Nice, this is going to make flashing for early users way easier, well, until I get to get preferences lib going :D

Reply:
yeah, i know it's gong to be phased out eventually - but i figured it would make early-use easier and more straight forward. Hoenstly - i'd just keep it there even with preferences lib, just in case the over wants to do a manual override in the code. They have a space to do that.

Reply:
We can do a check - if that value is anything other than the hard-coded value we switch to using that - otherwise we use the preferences lib.